### DISCUSSION TITLE
请问如何更新

### DISCUSSION THREAD

**Discussion by lyo623** [GaiZhenbiao/ChuanhuChatGPT#897] [Q&A] [answered]
点击检查更新后显示：
Local version check failed. But latest revision is [20230916]

按照手动更新的步骤操作，界面没有发生变化。

**Comment by Keldos-Li** [comment, marked as answer]
这是本地版本检查失败的提示。可能是因为你没有安装git。

**Comment by lyo623** [comment]
时间太久了 忘了我是docker部署的
